frontend/src/scenes/surveys/SurveyEditQuestionRow.tsx

```diff
@@ -125,6 +125,15 @@ export function SurveyEditQuestionGroup({ index, question }: { index: number; qu
         handleQuestionValueChange('descriptionContentType', key)
     }
 
+    const canHaveBranchingInput =
+        question.type === SurveyQuestionType.Rating ||
+        question.type === SurveyQuestionType.SingleChoice ||
+        question.type === SurveyQuestionType.MultipleChoice
```

Comment:
I think i have seen similar logic elsewhere, can we extract this to a method? so you just pass the question obj and return if it has branching or not

Reply:
maybe I saw in the SDK though

Reply:
made it a method